notebooks/guides/getting-started/v2/tutorial_pt6_v2.ipynb

```diff
@@ -0,0 +1,590 @@
+    "\n",
+    "Here, we show the most relevant documents with their similarity scores."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Rank: 1\n",
+      "Score: 0.34212792245283796\n",
+      "Document: {'text': 'Team-Building Activities: We foster team spirit with monthly outings and weekly game nights. Feel free to suggest new activity ideas anytime!'}\n",
+      "\n",
+      "Rank: 2\n",
+      "Score: 0.2883222063024371\n",
+      "Document: {'text': 'Proposing New Ideas: Innovation is welcomed! Share your brilliant ideas at our weekly team meetings or directly with your team lead.'}\n",
+      "\n",
+      "Rank: 3\n",
+      "Score: 0.278128283997032\n",
+      "Document: {'text': 'Joining Slack Channels: You will receive an invite via email. Be sure to join relevant channels to stay informed and engaged.'}\n",
+      "\n",
+      "Rank: 4\n",
+      "Score: 0.19474858706643985\n",
+      "Document: {'text': \"Finding Coffee Spots: For your caffeine fix, head to the break room's coffee machine or cross the street to the café for artisan coffee.\"}\n",
+      "\n",
+      "Rank: 5\n",
+      "Score: 0.13713692506528824\n",
+      "Document: {'text': 'Side Projects Policy: We encourage you to pursue your passions. Just be mindful of any potential conflicts of interest with our business.'}\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Compute dot product similarity and display results\n",
+    "n = 5\n",
+    "scores = np.dot(query_emb, np.transpose(doc_emb))[0]\n",
+    "max_idx = np.argsort(-scores)[:n]\n",
+    "\n",
+    "retrieved_documents = [faqs_long[item] for item in max_idx]\n",
+    "\n",
+    "for rank, idx in enumerate(max_idx):\n",
+    "    print(f\"Rank: {rank+1}\")\n",
+    "    print(f\"Score: {scores[idx]}\")\n",
+    "    print(f\"Document: {retrieved_documents[rank]}\\n\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Reranking can boost the results from semantic or lexical search further. The Rerank endpoint takes a list of search results and reranks them according to the most relevant documents to a query. This requires just a single line of code to implement.\n",
+    "\n",
+    "We call the endpoint using `co.rerank()` and pass the following arguments:\n",
+    "\n",
+    "- `query`: The user query\n",
+    "- `documents`: The list of documents we get from the semantic search results\n",
+    "- `top_n`: The top reranked documents to select\n",
+    "- `model`: We choose Rerank English 3\n",
+    "\n",
+    "Looking at the results, we see that the given a query about getting to know the team, the document that talks about joining Slack channels is now ranked higher (1st) compared to earlier (3rd).\n",
```

Comment:
There's a typo, how about:
"Looking at the results, we see that since the query is about getting to know the team, the document..."

Reply:
thanks!